lib/ratchet/component.js

```diff
@@ -39,9 +40,9 @@ function Component(template, name, channel) {
     render: function() {
       var data = this.state.data;
       if (Array.isArray(data)) {
-        return React.createElement('div', null, data.map(createComponent));
+        return React.createElement('div', null, data.map((d) => createComponent(template, d)));
```

Comment:
This is actually a _super_ nice thing about ES6 arrow functions, and how you can omit braces and `return` when it's a one liner. Haven't seen this before!